f3/certstore.go

```diff
@@ -0,0 +1,189 @@
+package f3
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
+
+	"github.com/Kubuxu/go-broadcast"
```

Comment:
I haven't looked at this library, but deps on individually-maintained things in high-value projects like this make me very nervous. Is the repo something appropriate to move into filecoin-project? Or if it's just a little code, can we copy-paste it into this one?

Reply:
It is 84 lines of code for something that @Stebalien and I looked for and could not find. It is a valuable library in general, but we can copy-paste it as well.